nh_eobs_mental_health/models/nh_clinical_wardboard.py

```diff
@@ -319,6 +320,7 @@ def read(self, cr, user, ids, fields=None, context=None,
                 elif spell.get('refusing_obs'):
                     rec['frequency'] = 'Refused - {0}'.format(rec['frequency'])
                     rec['next_diff'] = 'Refused - {0}'.format(rec['next_diff'])
+                elif spell.get('refusing_obs_blood_glucose'):
```

Comment:
May be missing from original code or not needed.

```suggestion
                elif spell.get('refusing_obs_blood_glucose'):
                    rec['bg_frequency'] = 'Refused - {0}'.format(rec['bg_frequency'])
```